lib/configuration.js

```diff
@@ -24,6 +24,7 @@ var Configuration = {
     metaData: {},
     logger: new Logger(),
     sendCode: true,
+    logAllDeliveryErrors: false,
```

Comment:
The name of this option might want changing to be consistent with the log message.